### PR TITLE
pure 1.15.0 (new formula)

### PR DESCRIPTION
**Pull request by danielbayley**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

**NOTE** that this `depends_on` #66167.

### CODE REVIEW

File: Formula/pure.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  bottle :unneeded
+
+  depends_on "zsh-async"
+
+  def install
+    zsh_function.install "pure.zsh" => "prompt_pure_setup"
+  end
+
+  def caveats
```
Comment (SMillerDev) [inline Formula/pure.rb:16]: This should be in the upstream documentation

File: Formula/pure.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class Pure < Formula
+  desc "Pretty, minimal and fast ZSH prompt"
+  homepage "https://github.com/sindresorhus/pure"
+  url "https://github.com/sindresorhus/pure/archive/v1.15.0.tar.gz"
+  sha256 "58915e0317763bae87f2c3b5b8e643d761dc64004e1153c8f5ba72f658c7fd74"
+  license "MIT"
+
+  bottle :unneeded
+
+  depends_on "zsh-async"
```
Comment (SMillerDev) [inline Formula/pure.rb:10]: No such formula exists

Comment (danielbayley) [reply Formula/pure.rb:10]: @SMillerDev As mentioned above… it `depends_on` #66167.

> **NOTE** that this `depends_on` #66167.

File: Formula/pure.rb (hunk 1 above)
Comment (danielbayley) [reply Formula/pure.rb:16]: @SMillerDev What do you mean?

**Comment by fxcoudert** [comment]
`zsh-async` was merged, please rebase on top of master, in a single commit titled `pure 1.15.0 (new formula)`

**Comment by danielbayley** [comment]
> `zsh-async` was merged, please rebase on top of master, in a single commit titled `pure 1.15.0 (new formula)`

@fxcoudert Done.

**Comment by SMillerDev** [comment]
Test failed on 10.14

**Pull request closed without merging**